src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/RequestSessionTemplateVisitStatsDto.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.dto.sessions
+
+import io.swagger.v3.oas.annotations.media.Schema
+import jakarta.validation.constraints.NotNull
+import java.time.LocalDate
+
+data class RequestSessionTemplateVisitStatsDto(
+  @Schema(description = "Visits from date stats", example = "2019-11-02", required = true)
+  @field:NotNull
+  val visitsFromDate: LocalDate,
```

Comment:
what if we allow a visitsToDate - nullable as well but default it to n days?

Reply:
not sure if we need that,...